snuba/datasets/storages/replays.py

```diff
@@ -25,6 +25,7 @@
             Array(UUID()),
         ),  # TODO: create bloom filter index / materialize column
         ("title", String(Modifiers(readonly=True))),
+        ("urls", Array(String())),
```

Comment:
Could you provide additional context on why this column needs to be an array?

Reply:
@nikhars It doesn't necessarily have to be an array.  We could write all the urls as separate replay record.  Is there a benefit to choosing one over the other?

Reply:
This became a moot point after a requirements change.